### ISSUE TITLE
[Component Update] Standalone checkbox and radio button should have rounded corners with Sosh and Wireframe themes

### ISSUE THREAD

**Issue by florentmaitre** [Orange-OpenSource/ouds-ios#1078]
### Description

The radius can be applied using ouds/💠_control/control-item/border/radius-item-only.

### Types of change

- [ ] Add new component
- [x] Update existing component

### Checklist for Core Team

- [ ] Tokens have been updated
- [ ] Documentations have been updated
- [ ] Unit tests have been updated
- [ ] UI tests have been updated
- [ ] Design System Toolbox has been updated
- [ ] Retrocompatibility has been checked, or broken API have been flagged as deprecated with a workaround

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.21.0**
- Build number: **1762294009**
- Build tag: **25f875c**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/0.21.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **0.21.0**
- Build number: **1762362134**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
